setup.py

```diff
@@ -20,9 +20,9 @@
 
 setup(
     name='wagtail_content_import',
-    version="0.1.6",
+    version="0.2.0",
     description='A module for Wagtail that provides functionality for importing page content from third-party sources.',
-    author='Samir Shah, Jacob Topp-Mugglestone, Karl Hobley, Matthew Westcott',
+    author='Samir Shah, Jacob Topp-Mugglestone,  Karl Hobley, Matthew Westcott',
```

Comment:
extra space

Reply:
Ah sorry - fixed just after committing, and recommitted!